### ISSUE TITLE
Mode collapse in training

### ISSUE THREAD

**Issue by Sherry-JQ** [KevinYuimin/StarGAN-Tensorflow#1]
I just trained the net as your "README.md" writes. While the samples are broken, and the loss is very big.
![008500](https://user-images.githubusercontent.com/28215086/39295311-a737abc2-4970-11e8-92a9-f83e32831497.jpg)
Maybe there are some problems about your parameters for training?
Can you help me?

**Comment by KevinYuimin** [comment]
Could you tell me more about your setting? Which attributes are you using? And also the adv_type and c_method

**Comment by Sherry-JQ** [comment]
All of my settings are default just as your ”main.py“. 
n_label is 6, attr_keys is "['Black_Hair','Blond_Hair','Brown_Hair', 'Male', 'Young','Pale_Skin']",
c_method is ”Sigmoid“, and adv_type is "WGAN". @KevinYuimin

**Comment by KevinYuimin** [comment]
I recheck the code, clone it, and run it again. It still works well.
This is my setting:
n_label is 5, attr_keys = ['Black_Hair','Blond_Hair','Brown_Hair', 'Male', 'Young'] 
c_method = ”Sigmoid“, adv_type ="WGAN", epoch = 10, batch_size=8, img_size=128. @Sherry-JQ
